security/pkg/pki/ca/generate_cert.go

```diff
@@ -79,14 +79,12 @@ func GenCSR(options CertOptions) ([]byte, []byte, error) {
 	// Generates a CSR
 	priv, err := rsa.GenerateKey(rand.Reader, options.RSAKeySize)
 	if err != nil {
-		glog.Errorf("RSA key generation failed with error %s.", err)
-		return nil, nil, err
+		return nil, nil, fmt.Errorf("RSA key generation failure (%s)", err)
```

Comment:
Same here, sometimes simply relying on 'err' itself can be hard to differentiate which line this function actually fails at.

Reply:
Done.